package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.1",
   "description": "Minimal web app",
   "main": "index.js",
+  "sideEffects": false,
```

Comment:
@mrozbarry What's this?

Reply:
Ah, just realized that was for [webpack 4](https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free). Apparently in the future, we can note which modules have side effects.